docs/support/ghc-version-support.md

```diff
@@ -19,6 +19,7 @@ Support status (see the support policy below for more details):
 |--------------|--------------------------------------------------------------------------------------|-----------------------------------------------------------------------------|
 | 9.6.2        | [latest](https://github.com/haskell/haskell-language-server/releases/latest)         | basic support                                                               |
 | 9.6.1        | [2.0.0.0](https://github.com/haskell/haskell-language-server/releases/tag/2.0.0.0)   | basic support                                                               |
+| 9.4.7        | [latest](https://github.com/haskell/haskell-language-server/releases/latest)         | full support                                                                |
 | 9.4.6        | [latest](https://github.com/haskell/haskell-language-server/releases/latest)         | full support                                                                |
 | 9.4.5        | [latest](https://github.com/haskell/haskell-language-server/releases/latest)         | full support                                                                |
```

Comment:
I think the entries for 9.4.5 and 9.4.6 are wrong also? They should say the last supported version was something previous.

Reply:
We ship binaries for `9.4.6` and `9.4.5` with the `2.2.0.0` release as well, so the table accurate afaict

Reply:
Hmm, I missed that. I thought the policy was to only ship binaries for the last minor release of each major release?

Reply:
Uhm, yeah, that's true, too.

Essentially, I thought it would be better to ship binaries for these two older minor releases as well, so people don't have to upgrade immediately, don't invalidate their entire cabal store for a minor release, but still get an updated HLS version. There was no additional overhead to shipping the other binaries, iirc.

Reply:
No worries, you know me, I just like there to be a policy so I know what we're doing :D